axelrod/strategies/meta.py

```diff
@@ -52,10 +56,48 @@ def __init__(self, team=None):
                     'manipulates_source',
                     'manipulates_state']:
             self.classifier[key] = any(t.classifier[key] for t in self.team)
-
         for t in self.team:
             self.classifier['makes_use_of'].update(t.classifier['makes_use_of'])
 
+    def __eq__(self, other):
```

Comment:
the problem with that (and the reason why `equality_attributes_test` in TestMeta was implemented the way it was I suppose, and that I reused in my `Meta.__eq__`) is that with this a `meta_player` and `meta_player.clone()` are not equal, which does not seems ideal either (and make all test_clone fail)
```
In [2]: p = axl.MetaPlayer()
In [3]: p2 = p.clone()
In [4]: p == p2
Out[4]: False
In [7]: for s, s2 in zip(p.team, p2.team):
   ...:     if s != s2:
   ...:         print(s)
   ...:         print(s2)
   ...:         
C
D
Arrogant QLearner
Arrogant QLearner
D
C
Cautious QLearner
Cautious QLearner
C
D
Hesitant QLearner
Hesitant QLearner
```
Chadys help me with english <3